doc/subsystems/networking/bsd-sockets.rst

```diff
@@ -0,0 +1,65 @@
+.. _bsd_sockets_api:
+
+BSD Sockets compatible API
+##########################
+
+Zephyr offers an implementation of a subset of BSD Sockets API (a part
+of POSIX standard). This allows to reuse existing programming experience
```

Comment:
Well, maybe this implementation, or this subset - keeping it underspecified was intentional, but ok.